holmes/plugins/toolsets/victorialogs/victorialogs.py

```
@@ -0,0 +1,774 @@
+) -> Optional[str]:
+    """Build a clickable VMUI link for a query."""
+    try:
+        base_url = (config.external_url or config.api_url).rstrip("/")
+        encoded_query = quote(query, safe="")
+        params = [f"query={encoded_query}"]
+        if start:
+            params.append(f"start={quote(start, safe='')}")
+        if end:
+            params.append(f"end={quote(end, safe='')}")
+        return f"{base_url}/select/vmui/?{'&'.join(params)}"
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)